serpent-moltres/fuel-column/postprocessing.py

```diff
@@ -9,7 +10,198 @@
 from auxiliary import plot_serpent_axial_collapse
 
 
-if __name__ == "__main__":
+def plotcsv_groups_collapse(file, save, lim, dire='z'):
```

Comment:
This postprocessing.py file is a problem. Reused functions should exist in one and only one place in the repo. No files should be copied, pasted, modified, and replicated across multiple files in a repo. Please review the basic reading in this group Best Practices in Scientific Computing.  https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1001745

Reply:
(one reason this is bad practice is that here I sit, re-reviewing a file that is *almost* identical to one I have reviewed already twice in a previous PR.)